octobot_trading/exchanges/util/exchange_market_status_fixer.pxd

```diff
@@ -16,25 +16,26 @@
 #  License along with this library.
 
 cpdef bint is_ms_valid(object value, bint zero_valid=*)
+cpdef bint check_market_status_limits(market_limit)
+cpdef bint check_market_status_values(market_limit, bint zero_valid=*)
 
 cdef void calculate_costs(dict market_limit)
 cdef void calculate_prices(dict market_limit)
 cdef void calculate_amounts(dict market_limit)
 cdef object get_markets_limit(dict market_limit)
 cdef void fix_market_status_limits_from_current_data(dict market_limit)
-cdef bint check_market_status_limits(market_limit)
-cdef bint check_market_status_values(market_limit, bint zero_valid=*)
 
 cdef class ExchangeMarketStatusFixer:
     cdef public object market_status
     cdef public object price_example
```

Comment:
I think it's a double

Reply:
And it shouldn't be public

Reply:
> I think it's a double

Don't forget the double :smile:

Reply:
With it being initialized bu default at None, it has to be object